### ISSUE TITLE
Bugfix in longwave radiaton thermal losses calculation 

### ISSUE THREAD

**Issue by gabriel-happle** [architecture-building-systems/CityEnergyAnalyst#1779]
@pajotca discovered a bug in the calculation of long wave radiation heat losses in the demand calculation.
He tried to address this in PR #1767.
This PR was declined. Instead PR #1778 was created to address the issue.

See more details in #1767 and #1778.

**Comment by martin-mosteiro** [comment]
Merged

**Comment by pajotca** [comment]
> 
> 
> @pajotca discovered a bug in the calculation of long wave radiation heat losses in the demand calculation.
> He tried to address this in PR #1767.
> This PR was declined. Instead PR #1778 was created to address the issue.
> 
> See more details in #1767 and #1778.

Just to understand, why was the PR declined?

And thank you for not assuming that contributors are inevitably men in the future.

**Comment by gabriel-happle** [comment]
Hi @pajotca,
The PR was declined, because according to my understanding your modification did not fix the bug in compliance with the ISO norm. (See discussion in #1767). 
I gave you suggestions to improve the fix, however I was not able to directly modify code in your fork.
Since you didn't react for some time, I just did it myself quickly.

Thank you for very much for reporting the bug!
Gabriel
